src/ImageSharp/MetaData/Profiles/Exif/ExifProfile.cs

```diff
@@ -233,11 +238,19 @@ public void SetValue(ExifTag tag, object value)
         /// <summary>
         /// Converts this instance to a byte array.
         /// </summary>
+        /// <param name="includeExifIdCode">Indicates, if the Exif ID code should be included.
+        /// This Exif ID code should not be included in case of PNG's. Defaults to true.</param>
         /// <returns>The <see cref="T:byte[]"/></returns>
-        public byte[] ToByteArray()
+        public byte[] ToByteArray(bool includeExifIdCode = true)
```

Comment:
I want to move the Exif identifier out of the ExifProfile as that's defined as part of the App1 segment itself in section 4.7.2 of the specification. 

http://www.exif.org/Exif2-2.PDF

Ideally we should pass the marker to the `ToByteArray` method when exporting. The `ExifProfile` constructor should have an overload accepting the marker.

Reply:
Fantastic, thanks!